playbooks/byo/openshift-cluster/upgrades/docker/docker_upgrade.yml

```diff
@@ -23,11 +23,23 @@
   serial: 1
   any_errors_fatal: true
   tasks:
-  - name: Prepare for Node draining
-    command: >
-      {{ openshift.common.client_binary }} adm manage-node {{ openshift.node.nodename }} --schedulable=false
+  - name: Load lib_openshift modules
+    include_role:
+      name: lib_openshift
+
+  - name: Mark node unschedulable
+    oadm_manage_node:
+      node: "{{ openshift.node.nodename | lower }}"
+      schedulable: False
     delegate_to: "{{ groups.oo_first_master.0 }}"
-    when: l_docker_upgrade is defined and l_docker_upgrade | bool and inventory_hostname in groups.oo_nodes_to_upgrade
+    retries: 10
+    delay: 5
```

Comment:
@mtnbikenc I'll defer to you on the best approach to take here, but I think that if we leave the retries in then we need at least two followup items to track:
- A bug to fix the upstream client bug
- A card/bug to track moving the workaround to the module, since we will still need to work against released versions of the oc/oadm client.